Resources/Prototypes/Entities/Markers/Spawners/Random/salvage.yml

```diff
@@ -31,12 +31,47 @@
         - CrateServiceBooks
         - CrateServiceSmokeables
         - CrateTrashCartFilled
+        - CrateChemistryP
+        - CrateChemistryS
+        - CrateChemistryD
+        - CrateEmergencyFire
+        - CrateEmergencyInflatablewall
+        - CrateEngineeringGear
+        - CrateEngineeringToolbox
+        - CrateEngineeringCableBulk
+        - CrateEngineeringElectricalSupplies
+        - CrateEngineeringJetpack
+        - CrateEngineeringMiniJetpack
+        - CrateEngineeringAMEJar
+        - CrateEngineeringSolar
+        - CrateFunPlushie
+        - CrateFunInstrumentsVariety
+        - CrateFunArtSupplies
+        - CrateFunBoardGames
+        - CrateFunATV
+        - CrateFunPirate
+        - CrateFunToyBox
+        - CrateFunMysteryFigurines
+        - CrateFunDartsSet
+        - CrateMedicalSupplies
+        - CrateChemistrySupplies
+        - CrateEmergencyBurnKit
+        - CrateEmergencyToxinKit
+        - CrateEmergencyO2Kit
+        - CrateEmergencyBruteKit
+        - CrateEmergencyAdvancedKit
+        - CrateEmergencyRadiationKit
```

Comment:
The medical crates are the only thing I have an issue with.  Having a place for medical ships is already a struggle, and having meds be this easy to find and collect only makes the problem worse.

Reply:
That's more than fair!

Reply:
Ahh. Didn't realize those didn't get automatically removed when I resolved the change. Removed em now!